book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(book, date)
+    Rental.new(book, date, self)
+  end
```

Comment:
- [ ] I'm not sure what went wrong with the previous review however in order for the `add_rental` method to work it should look like below:
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```
which is, as far as I can see, the way you had it before. 
I'm sorry for the confusion.